src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1753,6 +1755,21 @@ public ItemStack get(PlayerStonecutterRecipeSelectEvent event) {
 				}
 			}, EventValues.TIME_NOW);
 
+		if (Skript.isRunningMinecraft(1, 14)) {
+			EventValues.registerEventValue(PlayerTakeLecternBookEvent.class, ItemStack.class, new Getter<ItemStack, PlayerTakeLecternBookEvent>() {
+				@Override
+				public ItemStack get(PlayerTakeLecternBookEvent event) {
+					return event.getBook();
+				}
+			}, EventValues.TIME_NOW);
+			EventValues.registerEventValue(PlayerTakeLecternBookEvent.class, Lectern.class, new Getter<Lectern, PlayerTakeLecternBookEvent>() {
+				@Override
+				public Lectern get(PlayerTakeLecternBookEvent event) {
+					return event.getLectern();
```

Comment:
Correct me if I'm wrong, but is Lectern.class not a blockstate?
Skript doesn't use BlockStates.
Shouldn't this return the block instead?
```suggestion
			EventValues.registerEventValue(PlayerTakeLecternBookEvent.class, Block.class, new Getter<Block, PlayerTakeLecternBookEvent>() {
				@Override
				public Block get(PlayerTakeLecternBookEvent event) {
					return event.getLectern().getBlock();
```